docs-partials/builder/vsphere/common/FlagConfig-not-required.mdx

```diff
@@ -0,0 +1,10 @@
+<!-- Code generated from the comments of the FlagConfig struct in builder/vsphere/common/step_add_flag.go; DO NOT EDIT MANUALLY -->
+
+- `vbs_enabled` (bool) - Enable Virtualization Based Security option for virtual machine. Defaults to `false`.
```

Comment:
One quick question given the requirements needed for enabling vbs would having an example code block like in [vsphere-clone#vapp-options-configuration](https://developer.hashicorp.com/packer/integrations/hashicorp/vsphere/latest/components/builder/vsphere-clone#vapp-options-configuration) be helpful?

Reply:
I don't think it's necessary to be honest.